### ISSUE TITLE
DetailsList: In scan mode, Narrator is not reading "row selected" or "row unselected" on pressing Space

### ISSUE THREAD

**Issue by arunnagar15** [OfficeDev/office-ui-fabric-react#5163]
### Bug Report

- __Package version(s)__: (5.98.0)
- __Browser and OS versions__: ( browser- chrome, edge and windows 10 )

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (High) 

Products/sites affected: (Powerapps, cds-portal and BAF)

#### Describe the issue:

I have used the Details List did accessibility testing on it using Screen Reader (Narrator) and found the following issue,  When I am moving to Details List Row using tab and pressing Space , Narrator is reading Space only. I checked this issue in Microsoft Edge 41.16299.402.0  + Narrator in Scan Mode On.

#### Actual behavior:

Screen Reader (Narrator)  only reads  space when press space on Details List Row in Scan Mode on.

#### Expected behavior:

Screen Reader (Narrator)  should reads  Row Selected , Not selected when press space on Details List Row in Scan Mode on.

**Comment by ThomasMichon** [comment]
@arunnagar15 Narrator only reads selection toggle state if the focused element is of `role='checkbox'` it seems, but we need `role='row'` in order for the Accessible Grid to work correctly.
Do you think we might need to add a sub-element within the 'row' that actually receives focus and if of `role='checkbox'`, which is what you actually receive focus when navigating up and down in the list?

**Comment by eveleens** [comment]
Any update on this one?

**Comment by KevinTCoughlin** [comment]
Seems like a duplicate of #6525. A fix for that should be tested in scan mode as it will likely fix this issue as well.

**Comment by dzearing** [comment]
Closing as a dupe of #6525.
